### DISCUSSION TITLE
Automerge Major/Minor updates after approval by anyone.

### DISCUSSION THREAD

**Discussion by srmadsen** [renovatebot/renovate#15839] [Request Help] [answered]
I have automerge configured for patches but not major/minor updates.  How do I configure renovate to automerge major/minor after approval by anyone.

` "major": {
    "stabilityDays": 15,
    "automerge": false
  },
  "minor": {
    "stabilityDays": 10,
    "automerge": false
  },
  "patch": {
    "stabilityDays": 5,
    "automerge": true
  },
`

**Comment by rarkins** [comment, marked as answer]
No such feature exists
